### PR TITLE
Document return values for aws_s3

### PR DESCRIPTION
**Pull request by prasadkatti**
##### SUMMARY
This PR is for documenting the return values for `aws_s3`.

The output varies depending on the `mode` used. For e.g., the mode `get` returns only `msg` whereas the mode `put` return `msg` as well as `url`.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
`aws_s3` module

##### ANSIBLE VERSION

```
ansible 2.5.0
```

### CODE REVIEW

File: lib/ansible/modules/cloud/amazon/aws_s3.py [hunk 1]
```diff
@@ -236,6 +236,19 @@
...
+  description: msg indicating the status of the operation
+  returned: always
+  type: string
+  sample: PUT operation complete
+url:
+  description: url of the uploaded object
+  returned: (for PUT operation)
+  type: string
+  sample: https://my-bucket.s3.amazonaws.com/my-key.txt?AWSAccessKeyId=<access-key>&Expires=1506888865&Signature=<signature>
+'''
```
Comment (s-hertel) [inline lib/ansible/modules/cloud/amazon/aws_s3.py]: Mode getstr also returns `contents` in addition to msg.

**Comment by prasadkatti** [comment]
@s-hertel - Hi there. I believe that this is now ready for review. I have added all possible keys that might be returned.

**Comment by prasadkatti** [comment]
Hi. If I can get a comment or review on this one, I would be willing to add "return values" for some of the other modules which are missing them. :)

**Review by s-hertel — APPROVED** [review]

**Comment by willthames** [comment]
Merged, thanks @prasadkatti

**Pull request merged**